src/probnum/filtsmooth/statespace/integrator.py

```diff
@@ -0,0 +1,263 @@
+"""Integrated processes such as the integrated Wiener process or the Matern process.
+
+This is the sucessor of the former ODEPrior.
+"""
+try:
+    # cached_property is only available in Python >=3.8
+    from functools import cached_property
+except ImportError:
+    from cached_property import cached_property
+
+import numpy as np
+import scipy.special
+
+import probnum.random_variables as pnrv
+
+from . import discrete_transition, sde
+from .preconditioner import TaylorCoordinates
+
+
+class Integrator:
+    """
+    An integrator is a special kind of SDE,
+    where the :math:`i`th coordinate models the :math:`i`th derivative.
+    """
+
+    def __init__(self, ordint, spatialdim):
+        self.ordint = ordint
+        self.spatialdim = spatialdim
+
+    def proj2coord(self, coord: int) -> np.ndarray:
```

Comment:
this currently returns a matrix. I understand that building and multiplying a projection matrix with lots of zeros might be a bit slow, and that ideally, we either (i) implement a `ProjectionMatrix` operator which does this more efficiently (by implementing `__matmul__` or (ii) by writing a function that slices. I'd postpone this, though. What do you think?

Reply:
I'm fine with postponing :)